src/anonymous-credentials/credential-builder.ts

```diff
@@ -167,15 +170,15 @@ export class CredentialBuilder extends Versioned {
       s[STATUS_STR] = this._credStatus;
     }
 
+    if (!s['proof']) {
```

Comment:
Proof is generated after signing so why is the "type" of proof being included in what's supposed to be signed? Or are there other details about the proof that need to be known before signing?

Reply:
in the SDK there are other details required in proof. i would also argue that its important to sign each field in this lib's credential impl, including its type. probably makes sense to include a verification method and/or proof created dates in this impl too for signing (the SDK BBS+ credentials already have these fields and more)

Reply:
Ok.